### ISSUE TITLE
Some questions about the hyper-parameters?

### ISSUE THREAD

**Issue by ewrfcas** [julilien/PLDepth#1]
Thanks for the great work!
I have some questions about the hyper-parameter between the paper and the github implement.
1. The ranking size is 5 in the paper while the github code is 3.
2. The learning rate of PLDepth-EffNet is 1e-2,1e-3, or 1e-4? It is a little confused in the supplementary.
3. The best sampled ranking number is 100(supplementary) or 400(sec4.4.1), while the best factor is 5 in paper and codes.

**Comment by julilien** [comment]
Thanks for your interest.

1. Yes, you're right. The default ranking size parameter in the code is 3, but, however, 5 was used within the experiments. Thanks for pointing it out, I will update the default Click parameter accordingly.
2. For PLDepthEffNet, we have used an initial lr of 1e-4 (as opposed to 1e-3 for the ResNet version).
3. In Section 4.4.1, we used 400 rankings (of length 5) to provide a somewhat fair comparison to pairwise ranking with 2k sampled points per img and epoch (400 * 5 = 2000). For the other experiments, we used 100 rankings per img and epoch. We used more sampled rankings for the former experiment as we applied random-only sampling (as opposed to the depth ranking sampling described in Section 3.5 that was used for the latter part). But let me note that the model can certainly benefit from higher ranking sizes with larger numbers of samples when spending more compute budget (so it is not necessarily the best parameter).

**Comment by ewrfcas** [comment]
Thanks for the reply!
The default learning rate in codes is 1e-2. Should it be changed to 1e-4 for the PLDepthEffNet training with listwise MLE?

**Comment by julilien** [comment]
Yes, exactly.

**Comment by ewrfcas** [comment]
Thanks.
